packages/headless/src/api/service/case-assist/case-assist-params.ts

```diff
@@ -61,25 +61,22 @@ export const prepareSuggestionRequestFields = (
 ): CaseFields =>
   Object.keys(fields)
     .filter((fieldName) => fields[fieldName].value !== '')
-    .reduce(
-      (obj: CaseFields, fieldName) => ({
-        ...obj,
-        [fieldName]: fields[fieldName],
-      }),
-      {}
-    );
+    .reduce((obj: CaseFields, fieldName) => {
```

Comment:
There's an object mutation here (and in the `prepareContextFromFields` function below) that wasn't there before. I don't think its gonna cause any issue, but that's the only "eyebow raiser" I spotted in this PR.